File: 문제풀이/chap2-2/prob2-2-4.c
#include <stdio.h>

int N, A[109];
int Answer = 0;

int main() {
	// 입력
	scanf("%d", &N);
	for (int i = 1; i <= N; i++) scanf("%d", &A[i]);

	// 답 계산
	for (int i = 1; i <= N; i++) {
		Answer += A[i];
	}

	// 출력
	printf("%d\n", Answer % 100);
	return 0;
}

File: 문제풀이/chap2-2/prob2-2-4.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		// 입력
		int N = sc.nextInt();
		int[] A = new int[N + 1];
		for (int i = 1; i <= N; i++) {
			A[i] = sc.nextInt();
		}

		// 답 계산
		int Answer = 0;
		for (int i = 1; i <= N; i++) {
			Answer += A[i];
		}

		// 출력
		System.out.println(Answer % 100);
	}
};
